src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -230,15 +231,15 @@ public function countByChannel(ChannelInterface $channel)
     /**
      * {@inheritdoc}
      */
-    public function findLatestInChannel($count, ChannelInterface $channel)
+    public function findLatestInChannel(string $count, ChannelInterface $channel): array
```

Comment:
Here and in similar places - it's just a temporary solution to avoid invalid types exceptions (proposed solution: https://github.com/Sylius/Sylius/pull/8507)

Reply:
In this case, it's better not to typehint it at all (as passing int `10` will raise an error).